src/components/App.js

```diff
@@ -28,7 +28,7 @@ class App extends Component {
         <TopBar />
         <Grid container>
           <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-            <CalendarViewContainer />
+            <MainView />
```

Comment:
Let's rename this to `ClassesView` - I like that you realized `CalendarView` is no longer accurate, but specificity is always better than not!